packages/devtools_app/lib/src/provider/provider_screen.dart

```diff
@@ -0,0 +1,115 @@
+// Copyright 2021 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+import 'package:provider/provider.dart' as provider show Provider;
+
+import '../banner_messages.dart';
+import '../screen.dart';
+import '../split.dart';
+import './instance_viewer/instance_details.dart';
+import './instance_viewer/instance_providers.dart';
+import './instance_viewer/instance_viewer.dart';
+import 'provider_list.dart';
+
+final _hasErrorProvider = Provider.autoDispose<bool>((ref) {
+  if (ref.watch(providerIdsProvider) is AsyncError) return true;
+
+  final selectedProviderId = ref.watch(selectedProviderIdProvider).state;
+
+  if (selectedProviderId == null) return false;
+
+  final instance = ref.watch(
+    rawInstanceProvider(InstancePath.fromProviderId(selectedProviderId)),
+  );
+
+  return instance is AsyncError;
+});
+
+class ProviderScreen extends Screen {
+  const ProviderScreen()
+      : super.conditional(
+          id: id,
+          requiresLibrary: 'package:provider/',
+          title: 'Provider',
+          requiresDebugBuild: true,
+          icon: Icons.palette,
```

Comment:
feel free to change the icon - not sure if this once was listed in the conditional screens example by default.

Reply:
Done, switched to attach_file